apps/dashboard/src/components/workflow-editor/steps/utils/preview-context-storage.utils.ts

```diff
@@ -121,7 +121,7 @@ export function mergePreviewContextData(persistedData: ParsedData, serverDefault
   };
 }
 
-function mergeObjectData<T extends Record<string, unknown>>(persisted: T, serverDefault: T): T {
+export function mergeObjectData<T extends Record<string, unknown>>(persisted: T, serverDefault: T): T {
```

Comment:
Not related to this PR, but found a bug with mergin payload example from the server when the user added new keys to the schema